src/app/features/FeatureEditor/FeatureEditorController.js

```diff
@@ -35,6 +37,16 @@ var FeatureEditorController = function FeatureEditorController (
         featureFileStructure,
         featurePath
     );
+    /*runs a single feature*/
+    controller.runFeature = function(toRun){

```

Comment:
Move this function definition out of the constructor, i.e. `controller.runFeature = runFeature.bind(this);`